src/components/FilterInput.svelte

```diff
@@ -14,16 +16,20 @@
     margin-top: $spacing-xs;
     margin-bottom: $spacing-xs;
   }
-  input {
+  #filter-input {
     width: 100%;
+    cursor: pointer;
```

Comment:
Just noticed this. I think we should actually not do this, it's more common for search fields to just use the default pointer type.

```suggestion
```